RemoteInput/Plugin/Python/PythonPlugin.cxx

```diff
@@ -7,39 +7,125 @@
 //
 
 #include "PythonPlugin.hxx"
+#include "PythonCommon.hxx"
+#include "PythonShared.hxx"
 #include "PythonEIOS.hxx"
 #include "PythonJavaObject.hxx"
-#include "PythonReflectAPI.hxx"
 #include "Python.hxx"
+#include "NativePlugin.hxx"
 
 #include <unordered_map>
 #include <memory>
 
 std::unique_ptr<Python> python;
 
-// MODULE
+// MODULE 
 
 static struct PyMethodDef RemoteInputMethods[] =
 {
-        //{"load_module", nullptr, METH_VARARGS, "Documentation"},
-        {nullptr} /* SENTINEL */
+    // MARK: Non-EIOS
+    {(char*)"create_client", reinterpret_cast<PyCFunction>(PythonCreateClient), METH_FASTCALL, (char*)""},
+    {(char*)"_kill_zombie_clients", reinterpret_cast<PyCFunction>(PythonKillZombieClients), METH_FASTCALL, (char*)""},
```

Comment:
Do we need these to be in the module directly when they already exist on the objects?

Reply:
They don't *have* to, but I think I'd prefer if the all functions were available there. Is there a reason not to have them?